docs/source/user_guide/model_understanding.ipynb

```diff
@@ -179,6 +179,31 @@
     "graph_partial_dependence(pipeline, X, features='mean radius')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Two-way partial dependence plots are also possible and invoke the same API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "partial_dependence(pipeline, X, features=('worst perimeter', 'worst radius'))"
```

Comment:
I'm not sure why, but the two-way partial dependence calculation takes longer than the graph_partial_dependence function, which, iirc, calls partial_dependence.  It takes a fair amount of time...5 minutes? We would be accepting an increase to the build_docs runtime.

Reply:
I think if we set `grid_resolution=20` it should take a lot less time (it's what the `graph_partial_dependence` call below is doing.)

I believe this scales quadratically with the grid size in the two-way case and if so we should make a note of it here and/or the docstring.